content/general/concepts/qr-code/qr-code.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
We're missing "Subjects" and "Tags" from the metadata.  Look at the metadata in the [entry template](https://github.com/Codecademy/docs/blob/main/documentation/entry-template.md), and then look at [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) and [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) to see what to list there.